calour/filtering.py

```diff
@@ -422,6 +422,10 @@ def filter_abundance(exp: Experiment, min_abundance, **kwargs):
     min_abundance : numeric
         The minimal total abundance for each feature over all samples
 
+    Other parameters
```

Comment:
"Other parameters" is hard coded into docrep (part of the numpy doc standard)
Since docrep is just one file (the __init__.py), we can copy it into calour and then change it so it will also include kwargs parameters etc.
I think it will also be better to include it in calour since then we remove the dependency on a (very small) package that may change in the future without us knowing.
Alternatively, we can fork docrep and use the fork for installation (but then install will be via github... but that's also fine...).
or we can use the current docrep from its repository and keep it as Other Parameters...

what do you think?

Reply:
ok - let's copy it? if license allows

Reply:
license is currently GPL-2. I posted a question to the author if it's ok to copy/modify and use in our BSD-3 project. Let's hope he says it ok :).